### PR TITLE
Added logic to get connector doc link from FortiSOAR connector doc link

### PR DESCRIPTION
**Pull request by cs-ankit-gupta**
Description: Added logic to get connector doc link from FortiSOAR connector doc link

Fix:
1. Get connector doc link via connector title and add it to contents.md.

### CODE REVIEW

File: documenting/automation/solution-pack/SP-Doc-Automation/setup.py [hunk 1]
```diff
@@ -66,9 +67,27 @@ def __create_connector_data(self):
...
             return
 
+        connector_doc_data = self.__get_fortisoar_connector_doc_data()
+        logging.debug('Successfully fetched Fortinet FortiSOAR doc link')
         connector_data_path = self.sp_dir_path + '/connectors/data.json'
         connectors_data_json = json.load(open(connector_data_path))
         for connector in connectors_data_json:
             connectors_content += setup_connectors_data.substitute(connector_name=connector['label'],
-                                                                   connector_description=connector['description'], connector_doc_link='https://docs.fortinet.com/fortisoar/connectors') + '\n'
+                                                                   connector_description=connector['description'], connector_doc_link='https://docs.fortinet.com/fortisoar/connectors/{0}'.format(self.__get_connector_doc_link(connector_doc_data, connector['label']))) + '\n'
```
Comment (cs-suyog-jinturkar) [inline documenting/automation/solution-pack/SP-Doc-Automation/setup.py]: please const "FORTISOAR_CONNECTOR_DOC_LINK" here too

File: documenting/automation/solution-pack/SP-Doc-Automation/setup.py [hunk 2]
```diff
@@ -66,9 +67,27 @@ def __create_connector_data(self):
...
+    def __get_fortisoar_connector_doc_data(self):
+        response = requests.get(FORTISOAR_CONNECTOR_DOC_LINK)
+        return json.loads(response.text)
+
+    def __get_connector_doc_link(self, connector_doc_data, connector_label):
+        title_to_class = {item['title']: item['class']
+                          for item in connector_doc_data.get('connectors')}
+        if connector_label not in title_to_class.keys():
+            logging.debug(
+                'Doc link for connector - \"{0}\" has not been found.'.format(connector_label))
```
Comment (cs-suyog-jinturkar) [inline documenting/automation/solution-pack/SP-Doc-Automation/setup.py]: should be info or error, no?
Also, can we add more details about why its not found?

File: documenting/automation/solution-pack/SP-Doc-Automation/setup.py [hunk 3]
```diff
@@ -66,9 +67,27 @@ def __create_connector_data(self):
...
+            return ''
+        doc_link = title_to_class.get(connector_label)
+        logging.debug(
```
Comment (cs-suyog-jinturkar) [inline documenting/automation/solution-pack/SP-Doc-Automation/setup.py:91]: logging --> log (please change this)

File: documenting/automation/solution-pack/SP-Doc-Automation/setup.py (hunk 1 above)
Comment (cs-ankit-gupta) [reply documenting/automation/solution-pack/SP-Doc-Automation/setup.py]: incorporated

File: documenting/automation/solution-pack/SP-Doc-Automation/setup.py (hunk 2 above)
Comment (cs-ankit-gupta) [reply documenting/automation/solution-pack/SP-Doc-Automation/setup.py]: Should be info.
Not found if connector is of version 1.0.0 and docs and been not published yet.

File: documenting/automation/solution-pack/SP-Doc-Automation/setup.py (hunk 3 above)
Comment (cs-ankit-gupta) [reply documenting/automation/solution-pack/SP-Doc-Automation/setup.py:91]: Incorporated

**Review by cs-suyog-jinturkar — APPROVED** [review]

**Pull request merged**